src/EnergyPlus/HVACFourPipeBeam.hh

```diff
@@ -0,0 +1,280 @@
+#ifndef HVACFourPipeBeam_hh_INCLUDED
+#define HVACFourPipeBeam_hh_INCLUDED
+
+#include <memory>
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array1.hh>
+
+// EnergyPlus Headers
+#include <AirTerminalUnit.hh>
+#include <EnergyPlus.hh>
+#include <DataGlobals.hh>
+#include <GlobalNames.hh>
+#include <PlantLocation.hh> // this is from https://github.com/NREL/EnergyPlus/pull/4945  assuming it goes in other wise maybe the same from using DataPlant
+
+namespace EnergyPlus {
+
+namespace FourPipeBeam {
+
+class HVACFourPipeBeam : public AirTerminalUnit
+{
+
+public: // Creation
+
+	// Default Constructor
+	HVACFourPipeBeam() :
```

Comment:
I think if it is intended to only be used from within the factory, it would make more sense to keep it private.  The factory is needed to ensure you don't double instantiate things, so no reason to allow an external call to the constructor directly.  I'd recommend private at this point.  But I think (maybe?) there are further comments on this below?